GDPR.md

```diff
@@ -3,32 +3,30 @@ layout: page
 title: General Data Protection Regulation (GDPR)
 ---
 
-## GDPR-relevant information for and from joind.in
+## <abbr title="General Data Protection Regulation">GDPR</abbr>-relevant information for and from joind.in
 
-## Collected PII
+## Collected Personally Identifiable Information (<abbr title="Personally Identifiable Information">PII</abbr>)
 
-We collect the following mandatory information from users on registration
+We collect the following mandatory information from users on registration in order to identify the user.
 
 * Name
 * Username
-* Email-Address
+* Email Address
 
-These informations are necessary to identify the user.
+Additionally we collect the user's IP address when commenting on a talk. That is necessary because...??
 
-Additionally we collect the users IP-Address on Commenting. That is necessary because...??
+## Processes
 
-## Processes:
+### Deleting a user account
 
-### Deleting a user-account:
+If a user wants their user account to be deleted, the user sends the request with their username via
+the contact form on the website or via an email to gdpr@joind.in.
 
-If a user wants their user-account to be deleted the user sends the request with their username via
-the contact form on the website or via an email to gdpr@joind.in
+The team will then respond with an email to the user's email address that is registered in our database.
+After a confirmation the user's account will be deleted.
 
-The team will then respond with an email to the users email-address that is registered in our database. 
-After a confirmation the users account will be deleted.
-
-After removal all comments of that user will belong to an "anonymous user" and it will not be possible anymore to 
+After removal all comments of that user will belong to an "anonymous user" and it will not be possible anymore to
 trace them back to a certain user apart from information in the comments content. Comments will not be altered in this process.
 
-After removal talks will not be linked to a user-account any more but the users name will still be visible as the person 
-that gave that talk/session/keynote/etc as that is publicly available information from the Events schedule (printed or on their Website)
+After removal talks will not be linked to a user account any more but the user's name will still be visible as the person
+that gave that talk/session/keynote/etc since that is publicly available information from the Events schedule (printed or on their Website)
```

Comment:
```suggestion
that gave that talk/session/keynote/etc. since that is publicly available information from the event's schedule (printed or on their Website).
```

Reply:
@iansltx I'm sorry that I didn't commit this suggestion before pushing up the fix for the merge conflict. That ended the opportunity to merge this suggestion in. I took your suggestions and added them as a new commit: bb857ae